ftw/referencewidget/templates/handlebars.html

```diff
@@ -0,0 +1,41 @@
+<script id="refbrowser-template" type="text/x-handlebars-template">
+    <div class="refbrowser overlay">
+            <div class="pathbar">
+                <div class="path"></div>
+                <div class="search">
+                    <input class="searchField" name="refbrowser-search" type="text" />
+                    <button class="searchButton" name="refbrowser-search" type="submit">{{search}}</button>
+                </div>
+            </div>
+            <div class="listing">
+                <ul></ul>
+            </div>
+            <div class="formcontrols">
+                <div class="closecontainer">
+                 <button type="cancel" class="cancel">{{close}}</button>
+                </div>
+                <div class="batchingcontainer" />
+            </div>
+    </div>
+</script>
+
+<script id="node-template" type="text/x-handlebars-template">
+    <a data-path="{{path}}" href="#" data-clickable={{clickable}}>{{title}}</a>
```

Comment:
@jone The quotes are not necessary for valid markup --> https://mathiasbynens.be/notes/unquoted-attribute-values